modules/gdnative/SCsub

```diff
@@ -245,6 +245,14 @@ if ARGUMENTS.get('gdnative_wrapper', False):
     gd_wrapper_env = env.Clone()
     gd_wrapper_env.Append(CPPPATH=['#modules/gdnative/include/'])
 
+    if gd_wrapper_env['use_lto']:
+        if env.msvc:
```

Comment:
Did you test it? ;)

Reply:
(Just hinting that you got your logic backwards :))

Reply:
haha you're right I count on the CI to do the test for me on this one ;-)

...

Which tells me I should open an issue to enable gdnative wrapper compilation on CI don't you think ?

Reply:
aaaaaand push force on the PR to hide my mistake !